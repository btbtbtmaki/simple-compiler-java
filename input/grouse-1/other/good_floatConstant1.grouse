main {
	imm v := 1.2;
    imm w := v;
    imm x := -2.0;
    imm y := -123.456;
    imm z := 2.0e1;
    print v, w, x, y, z, nl;
}