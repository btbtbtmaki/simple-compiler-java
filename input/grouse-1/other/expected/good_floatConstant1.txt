1.2 1.2 -2 -123.456 20 
